src/orchestrator/orchestrator.rs

```diff
@@ -51,6 +53,9 @@ use crate::source::SourceCache;
 use crate::util::progress::ProgressBars;
 use crate::util::EnvironmentVariableName;
 
+const CONTAINER_ID_LENGTH: usize = 7;
+const JOB_UUID_LENGTH: usize = 36;
```

Comment:
Those magic constants should also be avoided. Do we even need them? Those IDs should have a fixed length anyway (or am I missing something) so why not rely on that? (We could theoretically provide re-exported or computed constants in the corresponding modules but it seems best to avoid it as long as we don't need them)